mgmt-hub/deploy-mgmt-hub.sh

```diff
@@ -818,19 +798,19 @@ else   # ubuntu and redhat
         tar -zxf $TMP_DIR/pkgs/$OH_ANAX_RPM_PKG_TAR -C $TMP_DIR/pkgs   # will extract files like: horizon-cli_2.27.0_amd64.rpm
         chk $? 'extracting pkg tar file'
         echo "Installing the Horizon agent and CLI packages..."
-        # Before install delete horizon-cli package if needed
-        PKG_NAME=horizon-cli
-        ${PKG_MNGR} list installed ${PKG_NAME} >/dev/null 2>&1
-        if [[ $? -eq 0 ]]; then
-            runCmdQuietly ${PKG_MNGR} ${PKG_MNGR_PURGE_CMD} ${PKG_NAME}
-        fi
-        # Before install delete horizon package if needed
-        PKG_NAME=horizon
-        ${PKG_MNGR} list installed ${PKG_NAME} >/dev/null 2>&1
-        if [[ $? -eq 0 ]]; then
-            runCmdQuietly ${PKG_MNGR} ${PKG_MNGR_PURGE_CMD} ${PKG_NAME}
-        fi
-        runCmdQuietly ${PKG_MNGR} ${PKG_MNGR_INSTALL_QY_CMD} $TMP_DIR/pkgs/horizon*.rpm
+        for pkg in horizon-cli horizon
+        do
+            PKG_NAME=${pkg}
+            ${PKG_MNGR} list installed ${PKG_NAME} >/dev/null 2>&1
```

Comment:
Wondering if you really need to check this. I initiated a slack discussion about it.